packages/data/README.md

```diff
@@ -1028,6 +1028,94 @@ function Component() {
 }
 ```
 
+## Selectors
+
+The following selectors are available on the object returned by `wp.data.select( 'core' )`.
+
+_Example_
+
+```js
+import { store as coreDataStore } from "@wordpress/core-data";
+
+function Component() {
+
+  const result = useSelect(
+
+    (select) => {
+
+      const query = { per_page: 20 };
+      const selectorArgs = ["postType", "page", query];
+
+      return {
+        pages: select(coreDataStore).getEntityRecords(...selectorArgs),
+        hasStartedResolution: select(coreDataStore).hasStartedResolution(
+          "getEntityRecords", // _selectorName_
+          selectorArgs
+        ),
+        hasFinishedResolution: select(coreDataStore).hasFinishedResolution(
+          "getEntityRecords",
+          selectorArgs
+        ),
+        isResolving: select(coreDataStore).isResolving(
+          "getEntityRecords",
+          selectorArgs
+        )
+      };
+    };
```

Comment:
```suggestion
    }
```
This semicolon breaks the script because it is before the closing paren of the useSelect call.